.travis.yml

```diff
@@ -1,12 +1,22 @@
-matrix:
+language: python
+dist: xenial
+python: 3.7
+node_js: latest
+before_install: pip install tox-travis
+install:
+  - yarn
+cache:
+  directories:
+    - $HOME/.cache/yarn
+jobs:
   include:
-    - language: node_js
-      cache: yarn
-      script:
-        - yarn lint
-    - language: python
-      dist: xenial
-      python: 3.7
-      install:
-        - pip install -r requirements.txt -e .
+    - name: JavaScript
+      script: yarn test
+    - name: Python
```

Comment:
Are you using Python 3.7?

Reply:
Yes. Should I be using a different version?

Reply:
No. That's good. I just wanted to confirm.